src/utils/index.ts

```diff
@@ -0,0 +1,48 @@
+export type OperationType='+' | '-' | 'X' | '/'|'='
+export type OperatorType='+' | '-' | 'X' | '/'
+const OPERATOR_FUNCTION={
+    ADD:function (a:number,b:number):number{
+        return Math.floor(a + b);
+    },
+    SUB:function (a:number,b:number):number{
+        return Math.floor(a - b);
+    },
+    MUL:function (a:number,b:number):number{
+        return Math.floor(a * b);
+    },
+    DIV:function (a:number,b:number):number{
+        return Math.floor(a / b);
+    },
```

Comment:
```suggestion
    ADD: (num1: number, num2: number) => Math.floor(num1 + num2),
    SUB: (num1: number, num2: number) => Math.floor(num1 - num2),
    MUL: (num1: number, num2: number) => Math.floor(num1 * num2),
    DIV: (num1: number, num2: number) => Math.floor(num1 / num2),
```
함수 반환 타입은 명시하지 않아도 자동으로 추론이 되다보니 제거해줘도 될 것 같아요~